Dockerfile

```diff
@@ -1,5 +1,10 @@
 FROM debian:stretch-slim
 
+# This script is downloaded from http://hg.nginx.org/pkg-oss/raw-file/default/build_module.sh
+ADD vendor/build_module.sh /
```

Comment:
As previously mentioned, move this below the retrieval of dependencies and prefer `COPY` by default: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy